### ISSUE TITLE
Add support for candles and modded torches

### ISSUE THREAD

**Issue by polykatt** [Tschipcraft/dynamiclights#38]
### Is your suggested enhancement related to a problem? Please describe.

I think candles should emit light when held in hand. Also wondering if there are any plans to support items from other mods such as Bone Torch and Aquatic Torches

### Describe the solution you'd like

Support for candles in all available colors and a few modded light sources

**Comment by Tschipcraft** [comment]
Yeah.. the wiki pages currently don't explain this well. I will update them soon.

Mod support is something that I don't want to maintain myself. I do plan on offering [mod_support tag lists](https://github.com/Tschipcraft/dynamiclights/tree/master/data/dynamiclights/tags/items/mod_support) that mod developers can hook into themselves, but I'm not sure if I want to keep the current format.

Candles don't emit light by default since they are not lit when placed down, which by my logic means that they are not lit when being held. One could argue though, that Steve is able to light torches out of thin air, so that would be a valid counterexample.
The other problem would be assigning them to a light level, since this kind of depends on how many candles a player is holding, which is something the pack currently doesn't check for.

**Comment by polykatt** [comment]
Ah okay yeah, that make sense then. Also no worry about mod support but I'll keep a look out if you add more information to the wiki. Thank you ^-^

**Comment by Tschipcraft** [comment]
I've updated the wiki pages now 🎉. If you need help with anything, feel free to reach out

**Comment by Tschipcraft** [comment]
I've changed my mind on this matter and decided to add native support for Bone Torch and Aquatic Torches. This will be part of the next release
